packages/wonder-blocks-icon-button/src/components/icon-button-core.tsx

```diff
@@ -149,7 +134,7 @@ const IconButtonCore: React.ForwardRefExoticComponent<
                 <StyledButton
                     type="button"
                     {...commonProps}
-                    disabled={disabled}
+                    aria-disabled={disabled}
```

Comment:
**note:** Changed from `disabled` to `aria-disabled` to match what we do with other clickable components (allowing to focus on disabled interactive elements to allow Screen Readers to detect these elements).

See: https://css-tricks.com/making-disabled-buttons-more-inclusive/